tests/FunctionalClientTest.php

```diff
@@ -273,7 +271,7 @@ public function testExecStreamCommandWithTtyAndStderrOutputWhileRunning($contain
         $stream->once('data', $this->expectCallableOnce('hello world'));
         $stream->on('end', $this->expectCallableOnce());
 
-        $output = Block\await(Stream\buffer($stream), Loop::get());
+        $output = \React\Async\await(Stream, Loop::get());
```

Comment:
Was getting rid of `\buffer($stream)` intentional? There's also a `Loop::get()` leftover.